avm/res/network/dns-forwarding-ruleset/main.bicep

```
@@ -25,7 +25,7 @@ param dnsForwardingRulesetOutboundEndpointResourceIds array
 param forwardingRules forwardingRuleType?
 
 @description('Optional. Array of virtual network links.')
-param vNetLinks array?
+param virtualNetworkLinks virtualNetworkLinkType?
```

```suggestion
param virtualNetworkLinks virtualNetworkLinkType
```

Not both the type and parameter must be nullable. One is enough 😉

---

Once that's done and the `Set-AVMModule` script run (you heard that right, it's a rap), I'm happy to approve

---

Done please rereview @AlexanderSehr